frontend/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
I think it would be better to have this gitignore merged with the one in `frontend/` and just ignore it like 
```
#.gitignore in the frontend folder
build/
```
This just ignores all `build` subdirectories everywhere recursively.